base/build.lua

```diff
@@ -155,7 +155,7 @@ indexstyle = "source2e.ist"
 
 -- Allow for TU and other test configurations
 checkconfigs = {"build","config-1run","config-TU","config-legacy","config-lthooks",
-                "config-lthooks2"}
+                "config-lthooks2","config-ltcmd"}
```

Comment:
My opinion is not too strong in favour of one or the other (neither do I see any downsides for either), but I went for a separate set of tests because it makes it much easier to “test changes to `ltcmd`”.  If I'm working on some change to `ltcmd` I can just run `l3build check -c config-ltcmd` and the relevant (or more likely to break) tests will be run.  Also because we also separated the tests for `lthooks`, which makes it easier if I want to change something specific to that module (of course that doesn't exempt me from running the full test suite, but I can triage first).